src/components/RightPane/CoursePane/CourseRenderPane.js

```diff
@@ -83,6 +89,18 @@ const flattenSOCObject = (SOCObject) => {
     }, []);
 };
 
+const RecruitmentBanner = ({ className }) => (
+    <div className={className}>
+        Interested in web development?
+        <br />
+        <a href="https://forms.gle/v32Cx65vwhnmxGPv8" target="__blank" rel="noopener noreferrer">
+            Join ICSSC and work on AntAlmanac and other projects!
+        </a>
+        <br />
+        We have positions for experienced devs and those with zero experience!
```

Comment:
Are there positions for zero experience?

Reply:
I was referring to the fellowship, but now I'm realizing it would be better worded as "opportunities" instead of "positions". Will change.